src/data/copy.json

```diff
@@ -66,6 +66,18 @@
       ],
       "contact": "Für weitere Fragen und stellvertretenden Kontakt, kontaktieren sie bitte: Solveig Rathenow <solveig.rathenow@finleap.com> "
     },
+    "press": {
+      "title": "PR Kontakt",
+      "pageHeader": "Schreiben Sie Uns",
+      "pressContactName": "Solveigh Rathenow",
+      "pressContactTitle": "Head of Venture Communications",
+      "pressContactEmail": "solveig.rathenow@finleap.com",
+      "pressContactPhone": "+49 176 321 44 053",
```

Comment:
didn't we said we take it out?

Reply:
right nobody told me. Taking phone number out

Reply:
done